_includes/selector.html

```diff
@@ -592,36 +557,33 @@
                 cmd += "</ul>";
                 return cmd;
             },
-            getImgTypeText(type) {
-                if (type === "core") return "Basic"
-                if (type === "standard") return "Basic w/ Dask-SQL"
-                return type;
-            },
-            getImgOptionText(option) {
-                if (option === "notebooks") return "Include Jupyter Notebooks";
-                if (option === "development") return "Development Image (only on Docker Hub)";
-                return option;
-            },
             getImgLoc(loc) {
                 if (loc === "NGC") return 'NGC (Stable Only) <i class="fa-regular fa-share-nodes"></i>';
                 if (loc === "Dockerhub") return 'Docker Hub (Stable and Nightly) <i class="fa-docker fab"></i>';
                 return loc;
             },
             getDockerNotes() {
-                var core_pkgs = this.rapids_meta_pkgs;
+                if (this.active_cuda_ver === "12.0") {
```

Comment:
Maybe write this more defensively for versions like 12.1?
```suggestion
                if (this.active_cuda_ver.startsWith("12")) {
```

Reply:
I like this! In `23.10` I think this logic isn't necessary anyway since we won't be including cuSignal anywhere, right? But it's good practice and I'll add it in.